### DISCUSSION TITLE
Display image based on URL [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2332] [Discord Threads]
> 🙋‍♂️ **djang0**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@889673993571950612> Currently, NocoDB displays images only if they are attached explicitly. URL based display isn't supported  
#  
> 🙋‍♂️ **djang0**  
#### oh ok thanks for your reply 😄  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/982020960192454766)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
